packages/apps/docs/src/docs/chainweb/compact-a-database.md

```diff
@@ -0,0 +1,99 @@
+
+   ```bash
+   Chainweb Tool
+
+   This executable contains misc commands that have been created for various
+   reasons in the course of Chainweb development. Linking executables is slow and
+   the resulting binaries are large, so it is more efficient in terms of build
+   time, space usage, download time, etc to consolidate them into one binary.
+   
+   Usage: cwtool COMMAND
+   
+   Available options:
+     -h,--help                Show this help text
+   
+   Available commands:
+     ea                       Generate Chainweb genesis blocks and their payloads
+     run-nodes                Run a local cluster of chainweb-node binaries
+     slow-tests               Run slow Chainweb tests
+     tx-list                  List all transactions in a chain starting with the most recent block
+     genconf                  Interactively generate a chainweb-node config
+     header-dump              Dump Block Headers to a JSON array
+     b64                      Command line utlis for Chainweb base64 encode/decode
+     db-checksum              Generate a checksum of all the checkpointer database tables between an inclusive range of blocks.
+     known-graphs             Encode know graphs as JSON values
+     tx-sim                   Simulate tx execution against real pact dbs
+     compact                  Compact pact database
+     pact-diff                Diff the latest state of two pact databases
+     calculate-release        Calculate next service date and block heights for upgrades
+   ```
+
+3. Create a backup copy of your current pact `sqlite` database to save all current state by running a command similar to the following:
+   
+   ```bash
+   cp -r /data/state/chainweb/db/0/sqlite /tmp/sqlite-backup-dd-mm-yyyy
+   ```
+   
+   Creating a backup copy of the database requires you to have more disk space available to store both the active database and the backup until you complete the compaction. 
+   However, creating a backup ensures that the node continues to run uninterrupted while the database is being compacted. 
+
+4. Compact your backup `sqlite` database by running the `cwtool compact` command with the following arguments:
+   
+   - `--target-blockheight` to keep enough state available to continue validating blocks after the database has been compacted. An appropriate value depends on the network your node is connected to. For example, if you are compacting your local development netowrk for testing purposes, you would set a lower block height than a node connected to the Kadena test network or main network. If this parameter isn't computed automatically, you should specify a value of at least 3.5 million (”3500000”) for a node connected to the Kadena main network.
+   - `--pact-database-dir` to specify the path to the backup pact `sqlite` state where all of the `*.sqlite`  files are located. The `cwtool compact` command creates one log file for each chain in this directory.
```

Comment:
The wording about the log files I find confusing. At first I read it to mean that the log files would go in this directory.  I realise now that's not what it means, but I don't really think logging is worth mentioning in this paragraph.